Added some blade directives to help in views

pralhadstha opened the pull request:
Tried to add some blade directives like:
set
explode
implode

Will add more if needed. Hope it would be helpful.

pralhadstha committed:
Added some blade directives to help in views

pralhadstha committed:
Added the missing include for Blade Facades

armababy commented:
Hmm. The `@set` one is good, but for  implode and explode i dont see benefit.
Could you please provide some use case or reasoning why `@explode()` over `{{ explode() }}`.

pralhadstha commented:
@armababy I agree with you `@explode`  performs same as `{{ explode() }}` it is helpful for cleaner views. If it is a overhead could probably remove it.

mikemand commented on Providers/CoreServiceProvider.php:
```diff
@@ -244,4 +247,50 @@ private function getCentralisedTranslationPath(Module $module)
+     */
+    private function getArguments($argumentString)
+    {
+        return explode(', ', str_replace(['(', ')'], '', $argumentString));
```
You need to be aware that in Laravel 5.3, Blade directives don't send along the outer parenthesis anymore.

Also, you could use [`str_getcsv`](http://php.net/manual/en/function.str-getcsv.php) to split the arguments:

``` php
return str_getcsv(mb_substr($argumentString, 1, mb_strlen($argumentString) - 2), ',', "'"); // mb_substr removes the beginning and ending parenthesis. Once we move to Laravel 5.3, this wont be neccessary; we could just pass $argumentString.
```

Or something of that nature.

nWidart replied:
Wasn't aware of that usage of `str_getcsv` in this case, interesting!  Maybe a good idea to use that, so that it's easily refactored on 5.3

nWidart commented:
Hello,

Thank you for your contribution!

I have the same remark as @armababy , I don't see a direct use case for explode/implode in views.
`@set` however can be very useful, twig has this as well.

pralhadstha committed:
Removed the explode and implode and modified argumentString for 5.2

pralhadstha commented:
Hi 
@nWidart and @armababy 
I have updated and removed the explode and implode with modifying the `argumentString` as @mikemand have suggested. Could you check and approve if you find it helpful.

nWidart commented:
Hey sorry for the late reply, merging now, thanks! 😄
